content/applications/general/email_communication/email_servers.rst

```diff
@@ -187,6 +187,44 @@ By default, inbound messages are fetched every 5 minutes in Odoo on-premise.
    :menuselection:`Settings --> Technical --> Automation --> Scheduled Actions` and look for
    :guilabel:`Mail: Fetchmail Service`.
 
+Set up different outgoing email servers for a multi-company environment
+=======================================================================
+
+The :guilabel:`FROM Filtering` field allows for the use of a specific outgoing email server
+depending on the :guilabel:`From` email address that Odoo is sending on behalf of. Access this
+field by navigating to :menuselection:`Settings --> Custom Mail Servers --> Outgoing Mail Servers
+--> New`.
+
+   .. image:: email_servers/from-filter-setting.png
+      :align: center
+      :alt: Outgoing email server settings and the FROM filter settings.
+
+When an email is sent from Odoo while the :guilabel:`FROM Filtering` field is set, an
+email server will then be chosen in the following sequence:
+
+#. First, Odoo searches for an email server having the same :guilabel:`FROM Filtering` value as the
+   :guilabel:`From` email address defined in the outgoing email. For example, if the
+   :guilabel:`From` email address is `test\@example.com`, only the email servers that have the
+   :guilabel:`FROM Filtering` value equal to `test\@example.com` will be returned.
+#. However, if no email servers are found using the :guilabel:`From` value, then Odoo searches for
+   an email server that has the same domain in the :guilabel:`FROM Filtering` field as the
+   :guilabel:`From` email address of the outgoing email. For example, if the :guilabel:`From`
+   email address is `test\@example.com`, only the email servers that have the :guilabel:`FROM
+   Filtering` value equal to `example.com` will be returned.
+
+If no email servers are found after checking for the domain, then Odoo will return all email
+servers that do not have any :guilabel:`FROM Filtering` value(s) set.
+
+Should this previous query return no results, then Odoo performs a search for an email server using
+the system parameter: `mail.default.from`. First, the email address listed will attempt to match an
+email server and then the domain will attempt a match. If no email server is found then Odoo will
+return the first outgoing email server (sorted by priority).
```

Comment:
```suggestion
the system parameter: `mail.default.from`. First, the email address listed attempts to match an
email server, and then the domain attempts to find a match. If no email server is found, Odoo
returns the first outgoing email server (sorted by priority).
```

Also clarify `the first outgoing email server` - is it the first one in the list on the `Outgoing Servers` page in Odoo?